docs/guides/software-RAID-SR.md

```diff
@@ -1,5 +1,11 @@
 # Software RAID Storage Repository
 
+:::warning
+Software RAID for storage repositories is documented here for our user community. However, it is not eligible to official support in a production environment.
```

Comment:
"eligible to official support" should be > "eligible for official support"

Reply:
thx, fixing